packages/labs/cli/src/test/uvu-wrapper.ts

```diff
@@ -0,0 +1,54 @@
+import * as uvu from 'uvu';
```

Comment:
Can do in a follow-up, but wonder if this is another thing that should maybe move to `tests/src` now that we have a consolidated place to put test utils?